rust/connlib/shared/src/messages.rs

```diff
@@ -26,6 +26,7 @@ impl ResourceId {
         ResourceId(Uuid::new_v4())
     }
 
+    #[cfg(feature = "proptest")]
```

Comment:
This isn't related to my PR at all, I just happened to notice it while working on the code in my VM. Cargo said this function isn't used, I figured it's for proptesting.